recipe/meta.yaml

```diff
@@ -11,14 +11,16 @@ source:
 
 build:
   number: 1
-  skip: True  # [win and py==38]
   binary_relocation: False  # [osx]
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  patches:
+    - 0001-force-cython.patch
```

Comment:
I think `patches` goes under `source` rather than `build`

Reply:
thanks! I should stop doing three things at the same time.